addons/vagrant/cumulus/Vagrantfile

```diff
@@ -249,6 +249,7 @@ vagrant_interface_rule
     device.vm.hostname = "node01" 
     
     device.vm.box = "debian/buster64"
+    device.vm.box_version = "10.20210228.1"
```

Comment:
Is there a reason that this one is not in the hosts file?

Reply:
Can we also move the cumulus vm.box_version in the hosts file?

Reply:
Done.

Reply:
Cool, then let's merge it.